bounties/pip/swig/1/vulnerability.json

```diff
@@ -0,0 +1,59 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-09-03",
+    "AffectedVersionRange": "*",
+    "Summary": "Code Execution",
+    "Contributor": {
+      "Discloser": "d3m0n-r00t",
+      "Fixer": ""
+    },
+    "Package": {
+      "Registry": "pip",
+      "Name": "swig",
+      "URL": "https://github.com/swig/swig"
+    },
+    "CWEs": [
+      {
+        "ID": "78",
+        "Description": "OS Commmand Injection"
+      }
+    ],
+    "CVSS": {
+      "Version": "3.1",
+      "AV": "",
+      "AC": "",
+      "PR": "",
+      "UI": "",
+      "S": "",
+      "C": "",
+      "I": "",
+      "A": "",
+      "E": "",
+      "RL": "",
+      "RC": "",
+      "Score": "7.5"
+    },
+    "CVEs": [""],
+    "Repository": {
+      "URL": "https://github.com/swig/swig",
+      "Codebase": ["Python"]
+    },
+    "Permalinks": ["https://github.com/swig/swig/blob/b018c32f9d0ba963560fa08da84802c23c41d89d/Tools/mkdist.py#L36",
```

Comment:
Inside the `permalinks` field is necessary inserting references to the `master` branch in order to avoid to index issues which are instead fixed in the `master`/main branch/latest version :smile:

Reply:
Hi @Mik317, 
I have added the change you asked. Hope its ok...
Thanks